tests/api/v1alpha2/tests.dhall

```diff
@@ -104,14 +104,27 @@ in  { accessSpec =
         , subnets = [ "example" ]
         }
       }
-    , instanceGroupSpec = Kops.InstanceGroupSpec::{
-      , role = "example"
-      , image = "example"
-      , minSize = 1
-      , maxSize = 1
-      , machineType = "example"
-      , subnets = [ "example" ]
-      }
+    , instanceGroupSpec =
+        let default =
+              Kops.InstanceGroupSpec::{
+              , role = "example"
+              , image = "example"
+              , minSize = 1
+              , maxSize = 1
+              , machineType = "example"
+              , subnets = [ "example" ]
+              }
+
+        in  { default
+            , withSysctlParameters =
+                    default
+                  ⫽ { sysctlParameters = Some
+                      [ "fs.pipe-user-pages-soft=524288"
+                      , "net.ipv4.tcp_keepalive_time=200"
+                      ]
+                    }
+                : Kops.InstanceGroupSpec.Type
+            }
```

Comment:
Amazing! The question is does it works?

Reply:
If it works we can return the `elasticsearch image` to it's origin state before I played with docker `entry`